ent-orm/usr/migrations/2_ent_migration.up.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE "users" ("id" bigint NOT NULL GENERATED BY DEFAULT AS IDENTITY, "age" bigint NOT NULL, "name" character varying NOT NULL DEFAULT 'unknown', PRIMARY KEY ("id"));
```

Comment:
Formatting suggestion for readability

```suggestion
CREATE TABLE "users"
(
    "id"   bigint            NOT NULL GENERATED BY DEFAULT AS IDENTITY,
    "age"  bigint            NOT NULL,
    "name" character varying NOT NULL DEFAULT 'unknown',
    PRIMARY KEY ("id")
);
```

Reply:
Ah, sorry ignore this - I saw the code above that generates this file 😄